src/amd.h

```diff
@@ -26,6 +26,14 @@ struct amd_drive {
 	int		initiator;
 };
 
+enum amd_led_interfaces {AMD_INTF_UNSET, AMD_INTF_SGPIO, AMD_INTF_IPMI};
+extern enum amd_led_interfaces amd_interface;
+
+enum amd_platforms {AMD_PLATFORM_UNSET, AMD_PLATFORM_ETHANOL_X,
+		    AMD_PLATFORM_DAYTONA_X, AMD_PLATFORM_GRANDSTAND,
+		    AMD_PLATFORM_SPEEDWAY};
```

Comment:
Please fix formatting of those enums. You can use enum ibpi_pattern as a reference.